numpy/core/fromnumeric.py

```diff
@@ -890,6 +890,13 @@ def argsort(a, axis=-1, kind='quicksort', order=None):
     array([[0, 1],
            [0, 1]])
 
+    >>> np.unravel_index(np.argsort(x, axis=None), x.shape)
+    (array([0, 1, 1, 0]), array([0, 0, 1, 1]))
+    >>> x[np.unravel_index(np.argsort(x, axis=None), x.shape)]
```

Comment:
I would probably stick to only the first line unless these additional examples show very different behavior -- space is at a premium in docstrings!